src/game/threat.ts

```diff
@@ -0,0 +1,13 @@
+import type { DeckSuit } from '../utils/constants';
+
+export interface Threat {
+  modal: boolean;
+  new: boolean;
```

Comment:
At some point, might it lead to problems to call this property the same as the `new` keyword? Should (and can be easily) rename this?

Reply:
yeah, it can be renamed, but that should be done separately.

Reply:
agreed.